our-app/src/components/authenticated/Dashboard.js

```diff
@@ -1,22 +1,23 @@
 import React from 'react';
-import { Card, ListGroup, Row, Col, Container} from 'react-bootstrap';
-import  ImmunizationList from '../common/ImmunizationList'
+import { connect } from 'react-redux';
+import { Card, ListGroup, Row, Col, Container } from 'react-bootstrap';
+import ImmunizationList from '../common/ImmunizationList'
 
-function Dashboard() {
+function Dashboard(props) {
     return (
-        <Container fluid="xl" style={{height: '80rem', width: '90rem'}}>
+        <Container fluid="xl" style={{ height: '80rem', width: '90rem' }}>
             <Row>
                 <Col>
                     <Card>
                         <Card.Header as="h1">Dashboard</Card.Header>
                         <ListGroup variant="flush">
                             <ListGroup.Item>
-                                <h2></h2>
+                                <h2>{props.name}</h2>
```

Comment:
@tletz24 is this where you wanted the user's name?

Reply:
Yes